Libraries/PushNotificationIOS/RCTPushNotificationManager.m

```diff
@@ -31,6 +31,11 @@
 
 NSString *const RCTErrorUnableToRequestPermissions = @"E_UNABLE_TO_REQUEST_PERMISSIONS";
 
+@interface RCTPushNotificationManager ()
+@property (nonatomic, copy) RCTPromiseResolveBlock requestPermissionsResolveBlock;
+@property (strong, nonatomic) NSMutableDictionary* remoteNotificationCallbacks;
```

Comment:
nit: our objc style is ` @property (nonatomic, strong) NSMutableDictionary *remoteNotificationCallbacks`